Automattic/jetpack
# VideoPress block: Adding new wpcom endpoint for updating VideoPress metadata.

#### Changes proposed in this Pull Request:
This PR adds a new `/wpcom/v2/videopress/meta` endpoint that we can utilize for future improvements of the VideoPress block to update metadata about the video.

~_Note: This currently won't work because of a bug on the wp.com api. Should be fixed soon._~ Fixed :)

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






 * Open a post in the block editor and open the browser console, and call this code for a VideoPress Id that the site has access to:
`wp.apiFetch( {
    path: '/wpcom/v2/videopress/meta',
    method: 'POST',
    data: { guid: {your-video-guid-goes-here}, rating: 'R-17' },
} ).then( res => {
    console.log( res );
} );`
 * If the request goes through, you should see `{code: "success", message: "Video meta updated successfully.", data: 200}` in the console and the age gate for your video when viewed.
 * Requests should fail with an object like `{code: "unknown_media", message: "The specified video was not found.", data: 404}` if the site does not have access to the video.

#### Proposed changelog entry for your changes:

* No changelog entry needed (I think?)
